app/modules/asset_groups/models.py

```diff
@@ -487,15 +513,15 @@ def check_all_job_status(self, all_scheduled):
                     'assuming Celery error and starting them again'
                 )
             try:
-                self.rerun_detection(background=False)
```

Comment:
This is now using a better default value, making this parameter not needed and results in the same behavior